<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-FAF54ES" failures="1" tests="2" name="Smoke" time="43.218" errors="0" timestamp="2022-06-28T14:34:15 PDT">
  <testcase classname="com.store.testcases.LoginPageTest" name="loginTest" time="4.892"/>
  <testcase classname="com.store.testcases.LoginPageTest" name="loginTest" time="2.543">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at com.store.testcases.LoginPageTest.loginTest(LoginPageTest.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Smoke -->
